protocol-designer/src/pages/Settings/index.tsx

```diff
@@ -52,6 +52,7 @@ export function Settings(): JSX.Element {
     : analyticsActions.optIn
 
   const prereleaseModeEnabled = flags.PRERELEASE_MODE === true
+  const pdVersion = process.env.OT_PD_VERSION?.split('-')[0]
```

Comment:
i think we can keep this as `process.env.OT_PD_VERSION`. it automatically updates to the latest tag. So right now it will say "-candidate-e" but that will update when we push the `8.2.0` tag.
```suggestion
  const pdVersion = process.env.OT_PD_VERSION
```